.travis.yml

```diff
@@ -36,6 +36,7 @@ env:
 
    - DESTINATION="OS=10.0,name=iPhone 6s" SDK="$IOS_SDK" SCHEME="$IOS_SCHEME" RUN_TESTS="YES" BUILD_EXAMPLE="YES" RUN_UI_TESTS="NO" POD_LINT="NO"
    - DESTINATION="OS=10.1,name=iPhone 6s" SDK="$IOS_SDK" SCHEME="$IOS_SCHEME" RUN_TESTS="YES" BUILD_EXAMPLE="NO"  RUN_UI_TESTS="YES" POD_LINT="NO"
+   - DESTINATION="OS=10.2,name=iPhone 6s" SDK="$IOS_SDK" SCHEME="$IOS_SCHEME" RUN_TESTS="YES" BUILD_EXAMPLE="NO"  RUN_UI_TESTS="YES" POD_LINT="NO"
```

Comment:
@jessesquires what's the rational for running UI/Unit tests on some targets but not others?